pkg/scheduler/api/resource_info.go

```diff
@@ -260,6 +260,15 @@ func (r *Resource) Sub(rr *Resource) *Resource {
 	return r.sub(rr)
 }
 
+// WithoutAssertSub subtracts two Resource objects without assertion.
+func (r *Resource) WithoutAssertSub(rr *Resource) *Resource {
+	ok, resources := rr.LessEqualWithResourcesName(r, Zero)
+	if !ok {
+		klog.Errorf("resources <%v> are not sufficient to do operation: <%v> sub <%v>", resources, r, rr)
```

Comment:
Here also need to `sub`.

Reply:
The judgment here is just to print the log, and the sub operation is executed uniformly when it finally return.

Reply:
Ok